src/3rdparty/libsvm/svm.cpp

```diff
@@ -729,6 +732,27 @@ void Solver::Solve(int l, const QMatrix& Q, const double *p_, const schar *y_,
 						G_bar[k] += C_j * Q_j[k];
 			}
 		}
+
```

Comment:
Since this is a custom edit to our bundled libsvm, should we also include some documentation in the readme (and maybe a patch?) detailing the difference between the released version and our version?